test/screenshot/lib/controller.js

```diff
@@ -233,14 +234,35 @@ class Controller {
     // https://github.com/request/request#requestoptions-callback
     return request({uri, encoding: null})
       .then(
-        (body) => body,
+        (body) => this.autoCropImage_(body),
         (err) => {
           console.error(`FAILED to download "${uri}"`);
           return Promise.reject(err);
         }
       );
   }
 
+  /**
+   * @param {!Buffer} imageData Uncropped image buffer
+   * @return {!Promise<!Buffer>} Cropped image buffer
+   * @private
+   */
+  async autoCropImage_(imageData) {
```

Comment:
Yes, exactly! It's cropping out the extra whitespace to make the images easier to view and diff.

Cropped (with this PR):
![image](https://user-images.githubusercontent.com/409245/39453801-7506a8dc-4c8c-11e8-876b-5d878ef68077.png)

Uncropped (without this PR):
<img src="https://user-images.githubusercontent.com/409245/39346129-60f89078-49a1-11e8-9315-4826dcdc3cc8.png" style="border: 1px solid black">

Reply:
Gotcha. My only concern with that is if the crop area changes but the screenshot does not which would count as a false negative. As long as Jimp's auto-cropping detection doesn't change, then it's probably fine.

Reply:
Ah, I see what you're saying. I think we're on the same page then 😁 

I put a 1px black border around the page content to try to avoid any problems with Jimp's auto-cropping detection.

The auto-cropping itself was specifically intended to avoid the situation where a browser's chrome changes (e.g., the omnibar gets taller), and suddenly all of our screenshot tests begin to fail out of nowhere.